docs/en/stack/security/authorization/privileges.asciidoc

```diff
@@ -75,6 +78,9 @@ node info, node and cluster stats, and pending cluster tasks.
 All read only {ml} operations, such as getting information about {dfeeds}, jobs,
 model snapshots, or results.
 
+`monitor_rollup`::
+All read only rollup operations. 
```

Comment:
"such as viewing the list of currently running jobs, and viewing rollup job capabilities of active and historical jobs."

Reply:
@polyfractal Should this be "viewing rollup capabilities of indices and index patterns" per the description of https://www.elastic.co/guide/en/elasticsearch/reference/master/rollup-get-rollup-caps.html?